lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,22 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "info"
+    notification_icon = "question"
+    notification_message = "Hint (1 of 2): A fruit is the fertilized ovary from a flower. Hint (2 of 2): A fruit contains seeds of the plant."
+    hint_notification = True
+%>
+
+<div class="notification ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
```

Comment:
@cptvitamin Hmmm, I was thinking that the message beside the icon would be sufficient (there will be a message for each type of event that brings up this notification area). Let's touch base about that and decide if we also want sr text.